README.md

```diff
@@ -16,6 +16,9 @@ This information can be used to RNG shiny and high IV Pokemon, similar to lua sc
 - Start + Select: Pause game and enable manual frame advancing
 - Select: Advance one frame while paused
 - Start or A: Unpause game
+- SOS Menu Commands:
+    - X + Right (D-Pad): Set SOS Caller slot to current Ally slot
```

Comment:
Let's make sure to print this in the SOS menu. Ideally people don't need to read the readme to know how to use PokeReader (which is a flaw pausing and advancing have 😅).